include/crow/json.h

```diff
@@ -36,6 +36,12 @@ namespace crow // NOTE: Already documented in "crow/app.h"
     {
         inline void escape(const std::string& str, std::string& ret)
         {
+            auto to_hex = [](char c) {
+                c = c & 0xf;
+                if (c < 10)
+                    return '0' + c;
+                return 'a' + c - 10;
+            };
```

Comment:
I was always thinking about moving out this into a small static inline function.

Reply:
take a look at the updated version, im not sure if it is desirable to have it kinda public since it is used only in this one spot but i don't have a real preference